src/items/defaultItems.ts

```diff
@@ -0,0 +1,85 @@
+import Item from './Item';
+import Dict from '../Dict';
+
+import * as Babylon from 'babylonjs';
+
+export default {
```

Comment:
At some point, we should associate this set of default items with a specific surface (Surface A), since other surfaces/boards will have other sets of items